docs/interpreting-experiments/confidence-intervals.md

```diff
@@ -1,34 +1,266 @@
+
+[^3]: Specifically: if the probability of it happening is less than the significance level that you have specified (which defaults to 5%).
+
+#### Pros of fixed-sample analysis
+
+- **Maximizes power for a given sample size.** If you have small sample sizes and struggle to get enough power to detect treatment effects, a fixed-sample experiment will give you the best shot at detecting an effect.
+
+#### Cons of fixed-sample analysis
+
+- **You have to select your sample size ahead of time.** Eppo's [Sample Size Calculator](../planning-experiments/index.md) makes this significantly easier, but if the assumptions you put into it end up not holding (for example, if the variance in your metrics is higher than it historically has been), then you can still end up underpowered at the predetermined sample size.
+- **You cannot decide to run the experiment to collect more sample**, even if you do end up with an underpowered experiment after reaching the predetermined sample size, or if, at that point, you decide you want to understand how the experiment affected different segments or subpopulations; doing so would increase your false positive rate above the specified significance level. Instead, you need to start a *new* experiment, and run it longer before making a decision.[^4]
+- **You cannot make decisions before the sample size is achieved** without invalidating the statistical guarantees.[^5] (aka: No peeking! 🙈) If you see a big metric drop, for example, and decide to shut down the experiment early, there is no guarantee that the chance it was a false positive (that is, that the drop was due to pure chance) is less than the significance level you've set. If you do this repeatedly, needlessly shutting down experiments that actually had neutral or even positive results. Similarly, if you want to ship experiments early when metrics seem to go up a lot, you'll end up shipping a lot of treatments with null or negative results.
+- **You cannot use only part of the data to make a decision.** If there's a data problem at a certain point in your experiment, so you cannot use the full sample size, you need to restart the experiment from scratch; trying to use the data you collected before the problem would cause the same issues as making a ship/shutdown decision before achieving the predetermined sample size. Similarly, if you find that the treatment effect is observed only for some segments of subjects, you cannot make a decision just on that subpopulation without obviating the significance level you've set.
```

Comment:
I don't really get this point -- if you throw away data without looking at outcomes (e.g. let's throw away all data on Tuesdays, ignoring weekly seasonality), then it should not impact the outcome?